cookbooks/bcpc-hadoop/attributes/yarn.rb

```diff
@@ -1,3 +1,46 @@
+require 'shellwords'
+
+default[:bcpc][:hadoop][:yarn][:env_sh].tap do |env_sh|
+  env_sh[:YARN_LOG_DIR] = '/var/log/hadoop-yarn'
+  env_sh[:YARN_PID_DIR] = '/var/run/hadoop-yarn'
+  env_sh[:YARN_IDENT_STRING] = 'yarn'
+  env_sh[:HADOOP_YARN_USER] = 'yarn'
+  env_sh[:YARN_CONF_DIR] = '$YARN_HOME/conf'
```

Comment:
Somehow this gets escaped into \$YARN_HOME/conf it should also not omit "${YARN_CONF_DIR:-$YARN_HOME/conf}"